src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -175,268 +176,310 @@ Engine::~Engine() {
-                                                          max_seq_len);
-        } else if (const auto &lstm_seq = std::dynamic_pointer_cast<const ngraph::opset6::LSTMSequence>(
-                node)) {
-            return lstm_seq->get_clip() == 0.0f &&
-                   lstm_seq->get_activations() == std::vector<std::string>{"sigmoid", "tanh", "tanh"} &&
-                   !ngraph::op::util::is_seq_len_provided(lstm_seq->get_input_node_shared_ptr(3),
-                                                          max_seq_len);
+        auto get_convert_precisions = []() {
+            precisions_array array = {
+                {ngraph::element::i64,     ngraph::element::i32},
+                {ngraph::element::u64,     ngraph::element::i32},
+                {ngraph::element::i16,     ngraph::element::i32},
+                {ngraph::element::u16,     ngraph::element::i32},
+                {ngraph::element::u32,     ngraph::element::i32},
+                {ngraph::element::f64,     ngraph::element::f32},
+                {ngraph::element::f16,     ngraph::element::f32},
+                {ngraph::element::boolean, ngraph::element::u8},
+                {ngraph::element::i4,      ngraph::element::i8},
+                {ngraph::element::u4,      ngraph::element::u8}
+            };
+
+            if (!dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core))
+                array.push_back({ngraph::element::bf16, ngraph::element::f32});
+
+            return array;
+        };
+        static const auto precisions = get_convert_precisions();
+
+        manager.register_pass<ngraph::pass::CommonOptimizations>();
+        manager.register_pass<ngraph::pass::WrapInterpolateIntoTransposes>();
+        manager.register_pass<ngraph::pass::TransposeSinking>();
+        manager.register_pass<ngraph::pass::ConvertSequenceToTensorIterator>();
+        manager.register_pass<ngraph::pass::ConvertOpSet3ToOpSet2>();
+        manager.register_pass<ngraph::pass::ConvertOpSet2ToOpSet1>();
+        manager.register_pass<ngraph::pass::ConvertTensorIteratorToSequence>();
+        manager.register_pass<ngraph::pass::LSTMCellDecomposition>();
+        manager.register_pass<ngraph::pass::GRUCellDecomposition>();
+        manager.register_pass<ngraph::pass::RNNCellDecomposition>();
+        manager.register_pass<ngraph::pass::ConvertNMS1ToNMS5>();
+        manager.register_pass<ngraph::pass::ConvertNMS3ToNMS5>();
+        manager.register_pass<ngraph::pass::ConvertNMS4ToNMS5>();
+        manager.register_pass<ngraph::pass::ConvertNMSToNMSIEInternal>();
+        manager.register_pass<ngraph::pass::ConvertMulticlassNmsToMulticlassNmsIE>();
+        manager.register_pass<ngraph::pass::ConvertMatrixNmsToMatrixNmsIE>();
+        manager.register_pass<ngraph::pass::TransposeMatMul>();
+        manager.register_pass<ngraph::pass::ConstantFolding>();
+
+        if (useLpt) {
```

Comment:
It seems we are not completely disabling lpt
I think we should set useLpt to false when disable Lpt transformation
For example using some additional marco, like
CPU_DEBUG_CAP_DISABLE_IF(useLpt, condition) or something similar.

Reply:
missed it, thank you, redesigned